rpm_spec/subpackages/manageiq-core

```diff
@@ -23,7 +23,9 @@ Requires: socat
 
 %pre core
 # ensure this user exists (build and upgrade)
-/usr/bin/id manageiq > /dev/null 2>&1 || /usr/sbin/useradd --system manageiq
+/usr/bin/id manageiq > /dev/null 2>&1 || /usr/sbin/useradd --system --create-home manageiq
+# create a manageiq home directory if it doesn't exist
+[[ ! -e /home/manageiq ]] && mkdir /home/manageiq && chown manageiq:manageiq /home/manageiq
```

Comment:
Is there a case where `--create-home` would not create the home directory?

Reply:
This second line is for upgrades

Reply:
I had this in here for upgrades where the user already exists but we don't have a homedir already.